hudi/README.md

```diff
@@ -0,0 +1,72 @@
+# Agent Check: Hudi
+
+## Overview
+
+This check monitors [Hudi][1].
+It is compatible with Hudi [versions][2] `0.10.0` and above.
+
+## Setup
+
+### Installation
+
+The Hudi check is included in the [Datadog Agent][3] package.
+No additional installation is needed on your server.
+
+### Configuration
+
+1. [Configure][4] the [JMX Metrics Reporter][5] in Hudi:
+
+    ```
+    hoodie.metrics.on=true
```

Comment:
I think it would be helpful to say that `hoodie` and `hudi` are referring to the same thing in this case.

Reply:
Users of Hudi would know this already, since it's widely used in their code and for all configuration options https://hudi.apache.org/docs/configurations#Read-Options